examples/connext_dds/dynamic_data_request_reply/c++11/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+# (c) 2022 Copyright, Real-Time Innovations, Inc. All rights reserved.
+# No duplications, whole or partial, manual or electronic, may be made
+# without express written permission.  Any such copies, or revisions thereof,
+# must display this notice unaltered.
+# This code contains trade secrets of Real-Time Innovations, Inc.
+cmake_minimum_required(VERSION 3.11)
+project(rtiexamples-dynamic-data-request_reply)
+set(CMAKE_MODULE_PATH
+    ${CMAKE_MODULE_PATH}
+    "${CMAKE_CURRENT_SOURCE_DIR}/../../../../resources/cmake"
+)
+
+# Find the RTI Connext DDS libraries
+find_package(RTIConnextDDS
+    "6.1.0"
```

Comment:
```suggestion
    "6.1.1"
```
Last but not least, in our master we have the `6.1.1` version already in place. Can you change this and merge master into your PR? Thanks in advance!